extra/gcloud-vm.ps1

```diff
@@ -0,0 +1,2 @@
+gcloud compute instances create dns-test --project=uptime-kuma-dev --zone=asia-east2-a --machine-type=e2-medium --network-interface=network-tier=PREMIUM,nic-type=GVNIC,stack-type=IPV4_ONLY,subnet=default --no-restart-on-failure --maintenance-policy=TERMINATE --provisioning-model=SPOT --instance-termination-action=STOP --service-account=787344463394-compute@developer.gserviceaccount.com --scopes=https://www.googleapis.com/auth/devstorage.read_only,https://www.googleapis.com/auth/logging.write,https://www.googleapis.com/auth/monitoring.write,https://www.googleapis.com/auth/servicecontrol,https://www.googleapis.com/auth/service.management.readonly,https://www.googleapis.com/auth/trace.append --tags=http-server,https-server --create-disk=auto-delete=yes,boot=yes,device-name=dns-test,image=projects/debian-cloud/global/images/debian-11-bullseye-v20230509,mode=rw,size=20,type=projects/uptime-kuma-dev/zones/asia-east2-a/diskTypes/pd-balanced --shielded-secure-boot --shielded-vtpm --shielded-integrity-monitoring --labels=goog-ec-src=vm_add-gcloud --reservation-affinity=any --key-revocation-action-type=stop
```

Comment:
Is it committed by mistake?

Reply:
Also extra/test-mariadb.ps1 and extra/test-push.ps1

Reply:
Oops...